tests/snap-run-symlink/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Check that symlinks to /usr/bin/snap trigger `snap run`
+prepare:
+    echo Ensure we have a os snap with `snap run`
+    sudo snap refresh --channel=beta ubuntu-core
+    sudo snap install hello-world
+restore:
+    sudo snap remove hello-world
```

Comment:
With the current `restore-each` content in `spread.yaml` we probably won't need to do snap-related cleanup. Otherwise, the test looks really nice! :)